src/main/resources/assets/betterendforge/lang/pt_br.json

```diff
@@ -0,0 +1,743 @@
+	"block.betterendforge.mossy_glowshroom_button": "Botão de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_chest": "Baú de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_crafting_table": "Bancada de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_door": "Porta de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_fence": "Cerca de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_gate": "Portão de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_ladder": "Escada de mão de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_log": "Haste de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_planks": "Tábuas de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_pressure_plate": "Placa de pressão de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_sign": "Placa de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_slab": "Laje de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_stairs": "Escadas de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_stripped_bark": "Volva de brilhomelo-musgoso descascada",
+	"block.betterendforge.mossy_glowshroom_stripped_log": "Haste de brilhomelo-musgoso descascada",
+	"block.betterendforge.mossy_glowshroom_trapdoor": "Alçapão de brilhomelo-musgoso",
+	"block.betterendforge.mossy_glowshroom_bookshelf": "Estante de livros de brilhomelo-musgoso",
+	
+        "block.betterendforge.end_lotus_bark": "Cepo de lótus-do-End",
+	"block.betterendforge.end_lotus_barrel": "Barril de lótus-do-End",
+	"block.betterendforge.end_lotus_button": "Botão de lótus-do-End",
+	"block.betterendforge.end_lotus_chest": "Baú de lótus-do End",
+	"block.betterendforge.end_lotus_crafting_table": "Bancada de lótus-do-End",
+	"block.betterendforge.end_lotus_door": "Porta de lótus-do-End",
+	"block.betterendforge.end_lotus_fence": "Cerca de lótus-do-End",
+	"block.betterendforge.end_lotus_gate": "Portão de lótus-do-End",
+	"block.betterendforge.end_lotus_ladder": "Escada de mão de lótus-do-End",
+	"block.betterendforge.end_lotus_log": "Tora de lótus-do-End",
+	"block.betterendforge.end_lotus_planks": "Tábuas de lótus-do-End",
+	"block.betterendforge.end_lotus_pressure_plate": "Placa de pressão de lótus-do-End",
+	"block.betterendforge.end_lotus_sign": "Placa de lótus-do-End",
+	"block.betterendforge.end_lotus_slab": "Laje de lótus-do-End",
+	"block.betterendforge.end_lotus_stairs": "Escadas de lótus-do-End",
+	"block.betterendforge.end_lotus_stripped_bark": "Cepo de lótus-do-End descascado",
+	"block.betterendforge.end_lotus_stripped_log": "Tora de lótus-do-End descascada",
+	"block.betterendforge.end_lotus_trapdoor": "Alçapão de lótus-do-End",
+	"block.betterendforge.end_lotus_bookshelf": "Estante de livros de lótus-do-End",
+    
+	"block.betterendforge.lacugrove_leaves": "Folhas de lacugrove",
+	"block.betterendforge.lacugrove_sapling": "Muda de lacugrove,
```

Comment:
Looks like you missed a " on this line :)

Reply:
glad that was just it, thank you for pointing it out